src/lib/modules/ESSelector/ESSelectorLoanRequest.js

```diff
@@ -36,7 +36,7 @@ export default class ESSelectorLoanRequest extends Component {
         }))
       : [];
     this.state['deliveryMethod'] = this.withDeliveryMethod
-      ? this.deliveryMethods[1].value
+      ? this.deliveryMethods[0].value
```

Comment:
I know it was like this in the past, but can you think of any other way of defining the delivery method than accessing the first list element? IMO this way will not be very reliable, order in config might change, especially when someone else configures and uses it